src/android/binding.rs

```diff
@@ -308,7 +308,7 @@ pub unsafe fn ipc(mut env: JNIEnv, _: JClass, arg: JString) {
 
       let arg = arg.to_string_lossy().to_string();
       if let Some(ipc) = IPC.get() {
-        (ipc.handler)(arg)
+        (ipc.handler)(IpcRequest { body: arg })
```

Comment:
we can also support this on android by storing a reference to the webview on `Ipc` class and getting the url from it